helsinkioppii/templates/tags/top_menu.html

```diff
@@ -2,7 +2,53 @@
 {% get_site_root as site_root %}
 
 {# Implementation of bootstrap theme dropdown menu - shows top level links and their children #}
-<nav class="navbar navbar-fixed-top navbar-default" role="navigation">
+
+<div class="navbar-helsinki navbar navbar-default navbar-static-top">
```

Comment:
The indentation is mismatched. Rest of the file uses 4 spaces and this block uses (mostly) 2 spaces. Code style should be consistent.

To be honest code style consistency is a project/repository wide problem. I'd suggest adding [editorconfig](http://editorconfig.org/) and proper linting. Eventually Travis should fail the build at linter errors. However it's too early at this point because the existing code base requires massive changes before all linter errors would be fixed.

Reply:
You are right. Projectwide linting has been on todo-list.
Do we have a standard style for html/django templates @jussiarpalahti?  

I can revert all changes on this to 4 spaces for now.

Reply:
@terotic, I use Pycharm which kind of takes care of these things for me... Though editor config would be a good addition.